storage/storage-interface/src/lib.rs

```diff
@@ -458,6 +458,15 @@ pub trait DbReader: Send + Sync {
         unimplemented!()
     }
 
+    /// Returns the proof of the given state key and version.
+    fn get_proof_by_version(
```

Comment:
naming: `get_state_proof()` -- because in the context of DbReader, "state" is not default.

Reply:
Done.